### ISSUE TITLE
users should be able to disable videos & animation

### ISSUE THREAD

**Issue by moates** [moates/onlyBans#12]
Given that many of our videos are overwhelming/distracting by design, you ought to be able to disable all videos at the beginning of the game. 

- [x] enable button on home page
- [x] obey browser requests to reduce motion? 

relevant resource 
https://alistapart.com/article/designing-safer-web-animation-for-motion-sensitivity/
